spec/requests/api/v1/forecast_location_spec.rb

```diff
@@ -9,26 +9,23 @@
     
     body = JSON.parse(response.body)
 
-    # set up VCR
-    # and actually assert on the values
-
     expect(body["data"]["attributes"]["current_weather_description"]).to eq("scattered clouds")
-    expect(body["data"]["attributes"]["current_temp"]).not_to be(nil)
-    expect(body["data"]["attributes"]["current_date_and_time"]).not_to be(nil)
-    expect(body["data"]["attributes"]["current_high"]).not_to be(nil)
-    expect(body["data"]["attributes"]["current_low"]).not_to be(nil)
-    expect(body["data"]["attributes"]["feels_like"]).not_to be(nil)
-    expect(body["data"]["attributes"]["humidity"]).not_to be(nil)
-    expect(body["data"]["attributes"]["visibility"]).not_to be(nil)
-    expect(body["data"]["attributes"]["uv_index"]).not_to be(nil)
-    expect(body["data"]["attributes"]["sunrise"]).not_to be(nil)
-    expect(body["data"]["attributes"]["sunset"]).not_to be(nil)
-    expect(body["data"]["attributes"]["hourly_time"]).not_to be(nil)
-    expect(body["data"]["attributes"]["hourly_temp"]).not_to be(nil)
-    expect(body["data"]["attributes"]["daily_weather_description"]).not_to be(nil)
-    expect(body["data"]["attributes"]["daily_precipitation"]).not_to be(nil)
-    expect(body["data"]["attributes"]["daily_high"]).not_to be(nil)
-    expect(body["data"]["attributes"]["daily_low"]).not_to be(nil)
+    expect(body["data"]["attributes"]["current_temp"]).to eq(72.99)
```

Comment:
originally I wanted to expect actual values but later learned without the tapes this would all fail